holidays/event.go

```diff
@@ -14,6 +14,8 @@ var location, lerr = time.LoadLocation(local)
 func init() {
 	if lerr != nil {
 		fmt.Println(lerr)
+		// 修复某些win机器报panic time missing Location in call to…错误
+		location = time.FixedZone("CST", 8*3600)
```

Comment:
CST 貌似是美国中部时间的缩写，用这个吧。

```suggestion
		location = time.FixedZone("UTC+8", 8*3600)
```